frontend/src/pages/org/workflow-editor.ts

```diff
@@ -174,7 +176,7 @@ const getDefaultFormState = (): FormState => ({
     minute: 0,
     period: "AM",
   },
-  runNow: false,
+  runNow: true,
```

Comment:
If you save a workflow with "Run on Save" disabled, the next time you go to edit the workflow it is always enabled. We don't have a way to persist "runNow" to the database to have it go back to the previous selection at this point, but might be worth adding in a follow-up PR?